exchanges/huobi/huobi_types.go

```diff
@@ -42,6 +42,23 @@ type DetailMerged struct {
 	Bid     []float64 `json:"bid"`
 }
 
+// Tickers contain all tickers
+type Tickers struct {
+	Data []Ticker `json:"data"`
+}
+
+// Ticker latest ticker data
+type Ticker struct {
+	Amount float64       `json:"amount"`
+	Close  float64       `json:"close"`
+	Count  int64         `json:"count"`
+	High   float64       `json:"high"`
+	Low    float64       `json:"low"`
+	Open   float64       `json:"open"`
+	Symbol currency.Pair `json:"symbol"`
+	Vol    float64       `json:"vol"`
```

Comment:
It's nice to have full name representation of struct fields, so Vol -> Volume would be ideal